docs/sources/panels/expressions.md

```diff
@@ -109,6 +109,18 @@ Log returns the natural logarithm of of its argument which can be a number or a
 
 The inf, nan, and null functions all return a single value of the name. They primarily exist for testing. Example: `null()`. (Note: inf always returns positive infinity, should probably change this to take an argument so it can return negative infinity).
 
+##### round
+
+The round function returns a rounded integer value. Example: `round(3.123)` or `round($A)`. (This function should probably take an argument so it can add presision to the rounded value).
```

Comment:
```suggestion
Round returns a rounded integer value. For example, `round(3.123)` or `round($A)`. (This function should probably take an argument so it can add precision to the rounded value).
```

Reply:
Changing the wording would then be inconsistent with how the inf, nan and null function wording has been done.

Reply:
We are refactoring docs and will remove the inconsistency. Use https://grafana.com/docs/grafana/latest/panels/expressions/#reduce as an example of how to add new content. Thanks.